items.c

```diff
@@ -123,14 +123,9 @@ void set_cas_id(uint64_t new_cas) {
 
 int item_is_flushed(item *it) {
     rel_time_t oldest_live = settings.oldest_live;
-    uint64_t cas = ITEM_get_cas(it);
-    uint64_t oldest_cas = settings.oldest_cas;
-    if (oldest_live == 0 || oldest_live > current_time)
-        return 0;
-    if ((it->time <= oldest_live)
```

Comment:
note to self: audit all callers to `item_is_flushed` to make sure they skip the item crawler and exptime == 0 stuff. they should, and the new line below is exactly what was moved to this function originally... but paranoia.

Reply:
all good